app/models/user.rb

```diff
@@ -67,7 +85,7 @@ def send_password_reset_email
   end
 
   def feed
-    Micropost.feed_user(id).recent_posts
+    Micropost.where("user_id IN (?) OR user_id = ?",following_ids, id).recent_posts
```

Comment:
1. tối ưu lại câu lệnh query
```suggestion
    Micropost.where(user_id: following_ids << id).recent_posts
```
2. nên khai báo where(user_id ...) thành scope trong model micropost để gọi ra trong này dùng nhé

Reply:
da